shared/fs/banner/reset-banner/index.tsx

```diff
@@ -92,12 +92,14 @@ const Banner = ({resetParticipants, onReAddToTeam, onViewProfile, onOpenWithoutR
         </Kb.Box2>
       ))}
     </Kb.Box2>
-    <Kb.Text type="BodySemibold" negative={true} style={styles.textOrUntil}>
-      Or until you're sure,{' '}
-      <Kb.Text type="BodySemiboldLink" negative={true} onClick={onOpenWithoutResetUsers}>
-        open a folder without {resetParticipants.length > 1 ? 'any of them' : 'them'}.
+    {resetParticipants.length > 1 ? (
```

Comment:
Not that I have a reason it's better per se, but I feel like normally we do `(bool) && (component)` instead of `(bool) ? (component) : null`

Reply:
@jakob223 changed